src/LocalDatastoreController.browser.js

```diff
@@ -40,14 +41,36 @@ const LocalDatastoreController = {
     const LDS = {};
     for (let i = 0; i < localStorage.length; i += 1) {
       const key = localStorage.key(i);
-      const value = localStorage.getItem(key);
-      LDS[key] = JSON.parse(value);
+      if (isLocalDatastoreKey(key)) {
+        const value = localStorage.getItem(key);
+        LDS[key] = JSON.parse(value);
+      }
     }
     return Promise.resolve(LDS);
```

Comment:
i'm confused still on where the promise is here?  why are you wrapping lds in a promise, why not just return LDS?

is it because this needs to have the same signature as another class of controller that does return promises?

Reply:
I just realized that flow typing isn't used in this project. I guess I just do it out of habit.

Reply:
:)

Reply:
I actually have flow for VS Code and it shows 300+ type errors in the project.